packages/aragon-wrapper/package.json

```diff
@@ -73,6 +73,7 @@
     "ethjs-ens": "^2.0.1",
     "homedir": "^0.6.0",
     "js-sha3": "^0.7.0",
+    "localforage": "^1.7.3",
     "lowdb": "^1.0.0",
```

Comment:
We could likely get rid of the `lowdb` and slim down our `lodash` requirements a lot!

Reply:
Cool, thanks for the comments. I'll change that and a few things Daniel pointed me